packages/client/src/forms/register/reviewReducer.ts

```diff
@@ -72,15 +72,32 @@ export const reviewReducer: LoopReducer<IReviewFormState, Action> = (
         reviewForm: {
           birth: {
             ...birth,
-            sections: [...birth.sections, review]
+            sections: [
+              ...birth.sections.filter(({ viewType }) =>
+                ['form', 'hidden', 'review'].includes(viewType)
+              ),
+              ...(birth.sections.find((s) => s.id === 'review') ? [] : [review])
+            ]
           },
           death: {
             ...death,
-            sections: [...death.sections, review]
+            sections: [
+              ...death.sections.filter(({ viewType }) =>
+                ['form', 'hidden', 'review'].includes(viewType)
+              ),
+              ...(death.sections.find((s) => s.id === 'review') ? [] : [review])
+            ]
           },
           marriage: {
             ...marriage,
-            sections: [...marriage.sections, review]
+            sections: [
+              ...marriage.sections.filter(({ viewType }) =>
+                ['form', 'hidden', 'review'].includes(viewType)
+              ),
+              ...(marriage.sections.find((s) => s.id === 'review')
+                ? []
+                : [review])
+            ]
```

Comment:
this adds the review section if it's not being defined? or am I understanding this correctly. Shouldn't the review section be there per the validation?

Reply:
ah right I was supposed to remove those after adding the validation. thanks for pointing out